OnePaySDK/QROnepayView.h

```diff
@@ -0,0 +1,18 @@
+//
+//  QROnepayView.h
+//
+//  Created by Rodrigo Ayala on 1/23/19.
```

Comment:
You should remove this kind of comment. Git has way more (and updated!) info about who did what.